jenkins/shared-library/vars/createMultibranchJob.groovy

```diff
@@ -53,6 +53,16 @@ void call(Map jobConfiguration, Boolean manualTrigger = true) {
 										strategyId(USE_CURRENT_SOURCE_STRATEGY_ID)
 									}
 
+									gitHubForkDiscovery {
+										strategyId(USE_CURRENT_SOURCE_STRATEGY_ID)
+										// One of the great powers of pull requests is that anyone with read access
+										// to a repository can fork it, commit some changes to their fork and then
+										// create a pull request against the original repository with their changes.
```

Comment:
feels like this comment is missing an ending 😅 e.g.

Since this can be abused, we don't want to run jenkins jobs for every pull request ...